### PR TITLE
updated post-vote release instructions

### PR DESCRIPTION
**Pull request by dgrove-oss**

### CODE REVIEW

File: docs/release_instructions.md [hunk 1]
```diff
@@ -162,22 +162,89 @@ restart the process with new candidate releases.  Update your
...
-
-  20. Announce the release -- must wait until website PR is merged and Jenkins publishes site.
-  30. Cleanup the artifacts from the release process:
-      a. Remove the rc files from staging.
-      b. Remove the previous If there is a previous released version, remove it from Apache release directory
-         (it will automatically still be available via the Apache archive server).
+### Publishing a Successful Release to Apache Dist Servers
+
+After a successful vote, the release manager will commit the artifacts
+being released to the openwhisk subdir of the Apache dist svn.
```
Comment (rabbah) [inline docs/release_instructions.md]: openwhisk
```suggestion
being released to the OpenWhisk subdir of the Apache dist svn.
```

File: docs/release_instructions.md [hunk 2]
```diff
@@ -162,22 +162,89 @@ restart the process with new candidate releases.  Update your
...
+
+The upload_to_dist.sh script automates the copy & svn add operations, but it
```
Comment (rabbah) [inline docs/release_instructions.md]:
```suggestion
The [`upload_to_dist.sh`](../tools/upload_to_dist.sh) script automates the copy & svn add operations, but it
```

i have to look up the path.

File: docs/release_instructions.md [hunk 3]
```diff
@@ -162,22 +162,89 @@ restart the process with new candidate releases.  Update your
...
+assumes that every file found in the release candidate subdir should be released.
+If this is not true (eg multiple parallel release votes), then the upload
```
Comment (rabbah) [inline docs/release_instructions.md]:
```suggestion
If this is not true (e.g., multiple parallel release votes), then the upload
```

File: docs/release_instructions.md [hunk 4]
```diff
@@ -162,22 +162,89 @@ restart the process with new candidate releases.  Update your
...
+
+Assuming the expected set of files were added, commit them:
+```
+cd ../stagingArea/svn_release && svn commit -m  "Apache OpenWhisk X.Y.Z release of <Component Description>"
+```
+
+Relatively soon after doing the svn commit, you should receive an email from
+from reporter.apache.org asking you to add your version data
+to its database.  Please follow the link and add the information
+(the data is used to generate periodic reports to the ASF Board).
```
Comment (rabbah) [inline docs/release_instructions.md]: does it make sense to include a sample email?

File: docs/release_instructions.md [hunk 5]
```diff
@@ -162,22 +162,89 @@ restart the process with new candidate releases.  Update your
...
+
+### Dockerhub updates
+
+If the components you released build docker images, then you should
+build the docker images locally, tag them with the release version
+(following the naming scheme for the repo), push the new images to
+dockerhub using the whiskbot dockerhub id, and update the `latest`
+tag to point to the new images.
+
+If you have published new images to dockerhub, submit PRs to openwhisk-deploy-kube
```
Comment (rabbah) [inline docs/release_instructions.md]:
```suggestion
If you have published new images to dockerhub, submit PRs to [openwhisk-deploy-kube](https://github.com/apache/openwhisk-deploy-kube)
```

File: docs/release_instructions.md [hunk 6]
```diff
@@ -162,22 +162,89 @@ restart the process with new candidate releases.  Update your
...
+and openwhisk-devtools (docker-compose) to use the new images.
```
Comment (rabbah) [inline docs/release_instructions.md]:
```suggestion
and [openwhisk-devtools (docker-compose)](https://github.com/apache/openwhisk-devtools) to use the new images.
```

File: docs/release_instructions.md [hunk 7]
```diff
@@ -162,22 +162,89 @@ restart the process with new candidate releases.  Update your
...
+
+### Update Downloads Page
+
+Submit a PR to openwhisk-website to update the Downloads page to
```
Comment (rabbah) [inline docs/release_instructions.md]:
```suggestion
Submit a PR to [openwhisk-website](https://github.com/apache/openwhisk-website) to update the Downloads page to
```

File: docs/release_instructions.md [hunk 8]
```diff
@@ -162,22 +162,95 @@ restart the process with new candidate releases.  Update your
...
+page (or some other URL that supports mirroring).  Do not include a direct
+link to the dist.apache.org svn server.
+
+Releases should always be announced to dev@openwhisk.apache.org.
+Releases can optionally be announced to announce@apache.org at the discretion
+of the release manager.
+
+### Post-release cleanup
+
+1. Remove the release candidate files from the staging svn
```
Comment (rabbah) [inline docs/release_instructions.md]:
```suggestion
1. Remove the release candidate files from the staging svn.
```

**Comment by dgrove-oss** [comment]
@rabbah - I think I managed to apply all the suggested updates; github was being a bit flaky and I had to do a few of them manually instead of accepting the proposed change.

**Pull request merged**